pkg/registry/apps/deployment/registry.go

```diff
@@ -65,7 +65,7 @@ func (s *storage) GetDeployment(ctx context.Context, deploymentID string, option
 }
 
 func (s *storage) CreateDeployment(ctx context.Context, deployment *extensions.Deployment, createValidation rest.ValidateObjectFunc) (*extensions.Deployment, error) {
-	obj, err := s.Create(ctx, deployment, createValidation, false)
+	obj, err := s.Create(ctx, deployment, createValidation, &metav1.CreateOptions{})
```

Comment:
Maybe make a metav1.CreateOptionsTODO() function so you can be sure to find and fix all of these after the fact? If you go this route, anyway.